src/GitVersion.Core/Extensions/NullableAttributes.cs

```diff
@@ -0,0 +1,149 @@
+// This was copied from https://github.com/dotnet/coreclr/blob/60f1e6265bd1039f023a82e0643b524d6aaf7845/src/System.Private.CoreLib/shared/System/Diagnostics/CodeAnalysis/NullableAttributes.cs
+// and updated to have the scope of the attributes be internal.
+
+#pragma warning disable CA1019 // Define accessors for attribute arguments
+
+namespace System.Diagnostics.CodeAnalysis
```

Comment:
Copy-paste all the attributes that will be available in new versions of dotnet. This class could be removed when dropping support to non-netcore fwk.